src/sticky.js

```diff
@@ -40,16 +42,16 @@ export default class Sticky extends React.Component {
   }
 
   componentDidMount() {
-    this.on(['resize', 'scroll', 'touchstart', 'touchmove', 'touchend', 'pageshow', 'load'], this.recomputeState);
+    this.on(['resize', 'scroll', 'touchstart', 'touchmove', 'touchend', 'pageshow', 'load'], () => this.recomputeState());
```

Comment:
Using an anonymous function here and in `componentWillUnmount` causes the events to never be removed and just added again over the top.